src/mse_lib_sgx/cli.py

```diff
@@ -170,6 +159,7 @@ def run() -> None:
         certificate=cert,
         uuid=args.uuid,
         need_ssl_private_key=ssl_app_mode == SslAppMode.CUSTOM_CERTIFICATE,
+        timeout=180,
```

Comment:
Set `180` in a global?

Reply:
Fixed in f438dd769553cbb6251098fce161bf95177ba429.